webserver/utilities.py

```diff
@@ -0,0 +1,18 @@
+import docx
+from sqlalchemy.orm import class_mapper
+
+def process_txt_file(uploaded_file):
+    content = uploaded_file.read()
+    return content.decode("utf-8")
+
+def process_docx_file(uploaded_file):
+    doc = docx.Document(uploaded_file)
+    text = [para.text for para in doc.paragraphs]
+    return '\n'.join(text)
+
+def serialise(model):
```

Comment:
This method's name is to general and does not do what it says (it doesn't write anything to disk)